crux-test/test/crux/api/TransactionInstantTest.java

```diff
@@ -91,7 +93,20 @@ public void mapIsFineWithExtraneousTest() {
 
         TransactionInstant tx = TransactionInstant.factory(map);
 
-        Assert.assertEquals(now, tx.getTime());
-        Assert.assertEquals(1L, (long) tx.getId());
+        assertEquals(now, tx.getTime());
+        assertEquals(1L, (long) tx.getId());
+    }
+
+    @Test
+    public void foo() {
```

Comment:
guessing this wasn't meant to be included? :wink:

Reply:
... I'mma go with no